website/source/api/secret/ad/index.html.md

```diff
@@ -191,6 +191,222 @@ $ curl \
+* `disable_check_in_enforcement` (bool: false, <optional>): Disable enforcing that service accounts must be 
+checked in by the entity or client token that checked them out. Defaults to false.
+
+When adding a service account to the library, Vault verifies it already exists in Active Directory.
+
+| Method   | Path                    |
+| :------- | :---------------------- |
+| `LIST`   | `/ad/library`           |
+| `POST`   | `/ad/library/:set_name` |
+| `GET`    | `/ad/library/:set_name` |
+| `DELETE` | `/ad/library/:set_name` |
+
+### Sample Post Request
+
+```
+$ curl \
+    --header "X-Vault-Token: ..." \
+    --request POST \
+    --data @payload.json \
+    http://127.0.0.1:8200/v1/ad/library/accounting-team
+```
+
+### Sample Post Payload
+
+```json
+{
+  "service_account_names": ["fizz@example.com", "buzz@example.com"],
+  "ttl": "10h",
+  "max_ttl": "20h",
+  "disable_check_in_enforcement": false
+}
+```
+
+### Sample Get Response
+
+```json
+{
+  "service_account_names": ["fizz@example.com", "buzz@example.com"],
+  "ttl": "10h",
+  "max_ttl": "20h",
+  "disable_check_in_enforcement": false
+}
+```
+
+### Sample List Response
+
+Performing a `LIST` on the `/ad/library` endpoint will list the names of all the sets of service accounts Vault contains.
+
+```json
+[
+  "accounting-team"
+]
+```
+
+## Check-out management
+
+These endpoints help manage check-outs.
+
+### Check a credential out
+
+Returns a `200` if a credential is available, and a `400` if no credential is available.
+
+* `name` (string: "", <required>): The name of the set of service accounts.
+* `ttl` (duration: "", <optional>): The maximum amount of time a check-out lasts before Vault 
+automatically checks it back in. Defaults to 24 hours. Setting it to zero reflects an unlimited lending period.
+Defaults to the set's `ttl`. If the requested `ttl` is higher than the set's, the set's will be used.
+
+| Method   | Path                              |
+| :------- | :-------------------------------- |
+| `POST`   | `/ad/library/:set_name/check-out` |
+
+### Sample Post Request
+
+```
+$ curl \
+    --header "X-Vault-Token: ..." \
+    --request POST \
+    --data @payload.json \
+    http://127.0.0.1:8200/v1/ad/library/accounting-team/check-out
+```
+
+### Sample Post Payload
+
+```json
+{
+  "ttl": "1h"
+}
+```
+
+### Sample Post Response
+```json
+{
+  "request_id": "364a17d4-e5ab-998b-ceee-b49929229e0c",
+  "lease_id": "ad/library/accounting-team/check-out/aoBsaBEI4PK96VnukubvYDlZ",
+  "renewable": true,
+  "lease_duration": 36000,
+  "data": {
+    "password": "?@09QW0KZ8DSBu3deIu7XLY1NZqzwhozmMAZ6v0IcZJGOjs5GvpVMvOeW7/duls2",
+    "service_account_name": "fizz@example.com"
+  },
+  "wrap_info": null,
+  "warnings": null,
+  "auth": null
+}
+```
+
+### Check a credential in
+
+By default, check-in must be called by the same entity or client token as was used for check-out.
```

Comment:
Maybe just:

By default, check-in must be called by the same entity or client token ~as was~ used for check-out.